Nightthor/main.py

```diff
@@ -0,0 +1,50 @@
+#Wprowadzenie danych -nazwa
+print ("Welcome")
+while True:
+    s = input ("Please input product name\n")
+    while not (1 <=len(s) < 60):
+        if not s:
+            print ("You didn't put any name\n")
+        elif len(s) >60:
+            print ("Incorrect name\n")
+        s = input ("Please input correct product name\n")
+    #Wprowadzenie danych -typ
+    word_list = ['medkit', 'helmet', 'rifle']
+
+    def get_word():
+        print("Please choose the type from the following:")
+        w_string = ""
+        for w in word_list:
+            w_string += w + ', '
+        print(w_string[:-1])
+
+        while True:
+            word = input("\n> ")
+            word = word.lower().strip()
+            if word in word_list:
+                return word
+            else:
+                print("'%s' is not in the list of words. Please choose from the following:\n%s" % (word, w_string[:-1]))
```

Comment:
your print is very old fashioned - new version of python allows better string interpolation
```suggestion
                print(f"'{word}' is not in the list of words. Please choose from the following:\n{w_string}")
```